File: Lista IV (JAVA) - Aula 10/J2_07/CD.java
public class CD extends Produto {
    private int faixas;

    CD(){}

    CD(String nome, double preco, int faixas){
        super(nome, preco);
        this.setFaixas(faixas);
    }

    public void setFaixas(int faixas) {
        this.faixas = faixas;
    }

    public int getFaixas() {
        return faixas;
    }

    public String toString(){
        return super.toString() + this.faixas + " faixas";
    }
}

File: Lista V (C++) - Aula 22/C1_06/CD.cpp
#include <iostream>
#include "CD.h"

//Construtor
CD::CD(const string& nome, double preco, int faixas) : Produto(nome, preco){
    this->faixas = faixas;
}

//Métodos
void CD::print(ostream &out) const {
	Produto::print(out); //chama a função de nome identico pertencente à produto
	out << " - " << faixas << " faixas"; //acrescenta a exclusividade de CD
}

ostream& operator<< (ostream &out, CD &cd) {
	cd.print(out);
	return out;
}